frontend/src/component/changeRequest/ChangeRequest/Changes/Change/StrategyChange.tsx

```diff
@@ -295,49 +345,7 @@ export const StrategyChange: FC<{
     return (
         <>
             {change.action === 'addStrategy' && (
-                <>
-                    <ChangeItemCreateEditDeleteWrapper>
-                        <ChangeItemInfo>
-                            <Typography
-                                color={
-                                    change.payload?.disabled
-                                        ? 'action.disabled'
-                                        : 'success.dark'
-                                }
-                            >
-                                + Adding strategy:
-                            </Typography>
-                            <StrategyTooltipLink
-                                name={change.payload.name}
-                                title={change.payload.title}
-                            >
-                                <StrategyDiff
-                                    change={change}
-                                    currentStrategy={currentStrategy}
```

Comment:
we were relying on the fact that current strategy is undefined because we look it up by change payload id that happens to be missing in the add strategy change payload. 
`'id' in change.payload && strategy.id === change.payload.id`
This is very difficult to reason about